src/components/GamesTill.vue

```diff
@@ -1,35 +1,34 @@
-<template>
-  <md-card class="md-layout-item md-size-30 md-small-size-100">
-    <md-card-header>
-      <div class="md-title" id="page-title">To Pass {{player.name}} ({{player.total}})</div>
-      <div class="md-subhead">{{player.total - lebron}} points</div>
-    </md-card-header>
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+export default class Gamestill extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      error: null,
+      loading: true
+    }
+  }
+  static propTypes = {
+    average: PropTypes.number,
+    player: PropTypes.object,
+    lebron: PropTypes.number
+  }
+  static defaultProps = {}
+  render() {
+    const difference = this.player.total - this.lebron
+    const gamesTill = Math.round(difference / this.props.average)
+    return (
+      <md-card className="md-layout-item md-size-30 md-small-size-100">
```

Comment:
So while the render part is returning the correct kind of content, the `md-*` objects won't work in react as they are vue components. So we would need to look into bringing in a new ui framework.